tests/runtest.sh

```diff
@@ -362,9 +338,6 @@ function create_core_overlay {
 
     cp -f -v "$coreFxBinDir/"* "$coreOverlayDir/" 2>/dev/null
     cp -f -p -v "$coreClrBinDir/"* "$coreOverlayDir/" 2>/dev/null
-    if [ -d "$mscorlibDir/bin" ]; then
```

Comment:
I have a feeling we may want to keep this option around for bringup targets such as arm(64) and linux x86 which do not build mscorlib natively yet.

Although I understand the desire to get rid of it.

Reply:
OK. I'll remain this option, but I think this implement should be changed since this option is not used to copy mscorlib now. I'll fix it.